pdk/BUILD

```diff
@@ -0,0 +1 @@
+# Required to be a package
```

Comment:
I think we should make sure to have the Apache license preamble in every file (this comment applies to other files as well)